build_venv.sh

```diff
@@ -4,22 +4,31 @@ module reset
 
 # create env
 # ---------
-ml cuda/11.3
+ml cuda/12.0
 ml cudnn
 ml nccl
 
-module load phdf5
-module load python3/3.9
+module load gcc/11.2.0
+module load intel/19.1.1
+module load intel/19.1.1
+module load intel/19.1.1
+module load intel/19.1.1
+module load impi/19.0.9
+module load mvapich2-gdr/2.3.7
+module load mvapich2/2.3.7
+
+module load phdf5/1.10.4
+module load python3/3.9.7
+
 export LD_LIBRARY_PATH=/usr/lib64:$LD_LIBRARY_PATH
```

Comment:
Since these two files are common. Could you move those to `modules.sh` file and `source modules.sh` from there. So we don't have to update in two different places?

Reply:
@kks32  applied the recommendation.